frontend/src/pages/detail/PODetail.tsx

```diff
@@ -15,15 +15,15 @@ import {
 } from "../../util/TableCellEditFuncs";
 import { useLocation } from "react-router-dom";
 
-interface PODetailState {
+export interface PODetailState {
```

Comment:
Why are we exporting these interfaces?

Reply:
We export them so that when passing information from SR/PO List, we know what state variables it needs.

We basically want to be able to pass the data and ensure the page doesn't crash